jaeger/jaeger_install/rhbjaeger-installation.adoc

```diff
@@ -0,0 +1,43 @@
+[id="installing-jaeger"]
+= Installing Jaeger
+include::modules/jaeger-document-attributes.adoc[]
+:context: jaeger-install
+
+toc::[]
+
+You can install Jaeger on {product-title} in either of two ways:
```

Comment:
Not sure we should mention about installing Jaeger as part of Service Mesh in the Jaeger section.

Reply:
It really needs to get mentioned somewhere that there are two different installation paths so that customers know they have to choose one or the other.